modules/tensor_mechanics/src/actions/DomainIntegralAction.C

```diff
@@ -828,8 +835,8 @@ DomainIntegralAction::act()
       _problem->addMaterial(mater_type_name, mater_name, params);
 
       std::string mater_name2;
```

Comment:
you could add a scope here wit `{` `}` to avoud creating a new unique variable name `mater_name2`

Reply:
This should have been addressed in the latest commit.